logr.go

```diff
@@ -447,9 +455,13 @@ func NewContext(ctx context.Context, logger Logger) context.Context {
 // LogSinks might want to know.
 type RuntimeInfo struct {
 	// CallDepth is the number of call frames the logr library adds between the
-	// end-user and the LogSink.  LogSink implementations which choose to print
+	// end-user and the LogSink. LogSink implementations which choose to print
 	// the original logging site (e.g. file & line) should climb this many
 	// additional frames to find it.
+	//
+	// The CallDepth is given for LogSink.Info and LogSink.Error such that 1 means
```

Comment:
Is this now fixed (Info -> enabled parallel to Enabled -> enabled)?

Reply:
Yes.

I have a test for this in https://github.com/kubernetes/klog/pull/383. That test fails with current logr and passes with this PR.

We can merge it now.

Reply:
Shouldn't we remove this part of the comment?